apps/client-server/src/app/form-generator/form-generator.service.spec.ts

```diff
@@ -87,17 +87,16 @@ describe('FormGeneratorService', () => {
           overrideDefault: false,
         },
         formField: 'description',
-        i18nLabel: 'form.descriptions',
-        label: 'Description',
+        label: 'description',
         row: 3,
         type: 'description',
       },
       rating: {
         col: 0,
-        defaultValue: SubmissionRating.ADULT,
+        defaultValue: 'ADULT',
         formField: 'rating',
-        label: 'Rating',
-        layout: 'vertical',
+        label: 'rating',
+        layout: 'horizontal',
```

Comment:
I will probably change this back to being vertical

Reply:
Done